dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -174,134 +174,143 @@ public ProcessingEnvironment getEnv() {
+                        {
+                            ConditionalBlock cb = (ConditionalBlock) b;
 
-                        // get store before
-                        TransferInput<A, S> inputBefore = getInputBefore(cb);
-                        TransferInput<A, S> input = inputBefore.copy();
+                            // get store before
+                            TransferInput<A, S> inputBefore = getInputBefore(cb);
+                            TransferInput<A, S> input = inputBefore.copy();
 
-                        // propagate store to successor
-                        Block thenSucc = cb.getThenSuccessor();
-                        Block elseSucc = cb.getElseSuccessor();
+                            // propagate store to successor
+                            Block thenSucc = cb.getThenSuccessor();
+                            Block elseSucc = cb.getElseSuccessor();
 
-                        propagateStoresTo(thenSucc, null, input, cb.getThenFlowRule(), false);
-                        propagateStoresTo(elseSucc, null, input, cb.getElseFlowRule(), false);
-                        break;
-                    }
+                            propagateStoresTo(thenSucc, null, input, cb.getThenFlowRule(), false);
+                            propagateStoresTo(elseSucc, null, input, cb.getElseFlowRule(), false);
+                            break;
+                        }
 
-                case SPECIAL_BLOCK:
-                    {
-                        // special basic blocks are empty and cannot throw exceptions,
-                        // thus there is no need to perform any analysis.
-                        SpecialBlock sb = (SpecialBlock) b;
-                        Block succ = sb.getSuccessor();
-                        if (succ != null) {
-                            propagateStoresTo(
-                                    succ, null, getInputBefore(b), sb.getFlowRule(), false);
+                    case SPECIAL_BLOCK:
+                        {
+                            // special basic blocks are empty and cannot throw exceptions,
+                            // thus there is no need to perform any analysis.
+                            SpecialBlock sb = (SpecialBlock) b;
+                            Block succ = sb.getSuccessor();
+                            if (succ != null) {
+                                propagateStoresTo(
+                                        succ, null, getInputBefore(b), sb.getFlowRule(), false);
+                            }
+                            break;
                         }
-                        break;
-                    }
 
-                default:
-                    assert false;
-                    break;
+                    default:
+                        assert false;
+                        break;
+                }
             }
-        }
 
-        assert isRunning;
-        isRunning = false;
+            assert isRunning;
```

Comment:
I would keep the assert and the update to `isRunning` together, as they are a logical unit.